.github/scripts/build_publish_nightly_docker.sh

```diff
@@ -3,13 +3,14 @@
 set -xeuo pipefail
 
 PYTORCH_DOCKER_TAG=$(git describe --tags --always)-devel
-CUDA_VERSION=11.3.1
+CUDA_VERSION=11.6.2
 
 # Build PyTorch nightly docker
 make -f docker.Makefile \
      DOCKER_REGISTRY=ghcr.io \
      DOCKER_ORG=pytorch \
      CUDA_VERSION=${CUDA_VERSION} \
+     CUDA_VERSION_SHORT="${CUDA_VERSION:0:2}.${CUDA_VERSION:4:1}" \
```

Comment:
I don't think this works as expected:
```
> export CUDA_VERSION=11.6.2
> echo ${CUDA_VERSION:0:2}.${CUDA_VERSION:4:1}
11..
```

I think the right way to do this would be as below, `echo $CUDA_VERSION | cut -d'.' -f 1-2`

Reply:
There's not even a need for this variable since we don't even use it

Reply:
Yes this is not needed variable. Removing!